build_deploy.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -ex
+#
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This script is executed by a Jenkins job for each change request. If it
+# doesn't succeed the change won't be merged.
+
+# make and rename uhc binary
```

Comment:
We are in the process of renaming `uhc` to `ocm` (for _OpenShift Cluster Manager_) so the less you use the ` uhc` word the better. So don't use it in comments, if possible.

Reply:
I was going to ask about this actually, good to know its already a WIP.